charts/gardener-extension-acl/values.yaml

```diff
@@ -3,13 +3,7 @@ imagePullPolicy: IfNotPresent
 
 replicaCount: 2
 
-resources:
-  requests:
-    cpu: "50m"
-    memory: "64Mi"
-  limits:
-    cpu: "100m"
-    memory: "128Mi"
+resources: {}
```

Comment:
I get the motivation for removing the limits. The option to not use limits if desired makes sense to me.
I'm wondering why you chose to remove the default requests though. VPA is enabled by default but doesn't work without requests set.
I would expect that it makes most sense to have some sensible default requests.

Reply:
The only argument to set resources to an empty map is that it's consistent to other extensions.
But I agree, that it makes sense to have defaults here.
I will add the requests again.